blog/announcements.xml

```diff
@@ -1,6 +1,125 @@
+        <a href="https://github.com/fridh">Frederik Rietdijk</a>,
+        <a href="https://github.com/supersandro2000">Sandro Jäckel</a>,
+        <a href="https://github.com/nequissimus">Tim Steinbach</a>,
+        <a href="https://github.com/jonringer">Jonathan Ringer</a>.
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/marsam">Mario Rodas</a>,
+        <a href="https://github.com/dotlambda">Robert Schütz</a>,
+        <a href="https://github.com/jtojnar">Jan Tojnar</a>,
+        <a href="https://github.com/sternenseemann">Sterni</a>,
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">in the top three by total number of packages</a>.
+      </p>
+
+      <p>
+        This didn't stop us. In the last six months:
+      </p>
+
+      <ul>
+        <li>12985 packages were added</li>
+        <li>14109 packages were removed</li>
+        <li>16768 packages were updated</li>
+      </ul>
+
+      <p>
+        Likewise, our NixOS module system got bigger and better:
+      </p>
+
+      <ul>
+        <li>1527 options were added</li>
+        <li>290 options removed</li>
+        <li>400 options were updated</li>
+      </ul>
+
+      <h3>Stabilization Contributors</h3>
+
+      <p>
+        Stabilization of the NixOS happens a month before the planned release. The
+        goal is to have as little as possible continuous integration (Hydra)
+        jobs failing before the release is cut.
+      </p>
+
+      <p>
+        Individuals who contributed to stabilizing this release:
+        <a href="https://github.com/fabaff">Fabian Affolter</a>,
+        <a href="https://github.com/sternenseemann">Sterni</a>,
+        <a href="https://github.com/stephank">Stéphan Kochen</a>,
+        <a href="https://github.com/dotlambda">Robert Schütz</a>,
+        <a href="https://github.com/mweinelt">Martin Weinelt</a>,
+        <a href="https://github.com/jonringer">Jonathan Ringer</a>.
+        <a href="https://github.com/alyssais">Alyssa Ross</a>,
+        <a href="https://github.com/thefloweringash">Andrew Childs</a>,
+        <a href="https://github.com/ttuegel">Thomas Tuegel</a>,
+        <a href="https://github.com/maralorn">Malte Brandy</a>, and 431 others!
+      </p>
+
+      <h3>Special Thanks</h3>
+      <p>
+        I would like to give a special thanks to <a href="https://github.com/jtojnar">Jan Tojnar</a> and others for the Gnome 40 stabilization effort. Another special thanks should be given to <a href="https://github.com/ttuegel">Thomas Tuegel</a> and many others for bringing Plasma 5.21 to NixOS.
+      </p>
+
+      <h3>Reflections and Closing</h3>
+
+      <p>
+        I think the [RFC80](https://github.com/NixOS/rfcs/pull/80) and [RFC85](https://github.com/NixOS/rfcs/pull/85)
```

Comment:
:warning: Markdown links!

Reply:
yea, I just noticed and fixed that myself  :) Thanks though!